src/AHitbox.hpp

```diff
@@ -0,0 +1,23 @@
+/*
+** EPITECH PROJECT, 2024
+** OpenthisFuckinDoor
+** File description:
+** AHitbox
+*/
+
+#include "AEntity.hpp"
+
+class AHitbox:public AEntity
+{
+public:
+    AHitbox(float x, float y);
+    ~AHitbox() = default;
+    
+    std::pair<float, float> getHitboxSize() {return (_hitboxSize);}
```

Comment:
I think this getter is useless because the only utility of hitboxes are the setter to set the wanted hitbox, and the isTouched method to trigger a potential event